samples/nrf9160/modem_trace_flash/src/trace_flash.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stddef.h>
+
+/**
+ * @brief Flush traces to flash
+ *
+ * @details Forces write of the remaining block buffer to flash. Call this function after modem
+ * traces has been turned off and no more trace data is expected.
+ *
+ * @return 0 on success, negative errno code on fail
+ */
+int trace_flash_flush(void);
+
+/**
+ * @brief Read trace data from flash
+ *
```

Comment:
Add a note that this function shall be only called after all trace writing operations are complete. 
(Because we dont protect the `traces_size` parameter).